lib/rspec-rails.rb

```diff
@@ -67,6 +73,7 @@ def supports_action_mailer_previews?(config)
         # always return `true`.
         config.respond_to?(:action_mailer) && ::Rails::VERSION::STRING > '4.1'
       end
+
```

Comment:
Would you remind removing this? Adding extra spacing to existing code tends to cause git churn which we would like to avoid.